glean-core/src/metrics/remote_settings_config.rs

```diff
@@ -31,6 +31,12 @@ pub struct RemoteSettingsConfig {
     /// enabled state of the ping of the same name.
     #[serde(default)]
     pub pings_enabled: HashMap<String, bool>,
+
+    /// This is a `u32` representing the threshold of events that
+    /// will be buffered before an events ping is collected and
+    /// submitted.
```

Comment:
I see we documented the others similar but I think this doesn't need repeating the type and something like this reads more naturally:

```suggestion
    /// This is the threshold of events that will be buffered before an events ping is collected and
    /// submitted.
    /// It overrides the value configurated at initialization time.
```
(added one more sentence too)